src/Compilers/VisualBasic/Portable/Microsoft.CodeAnalysis.VisualBasic.vbproj

```diff
@@ -73,9 +73,6 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.EditorFeatures2.UnitTests" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="My Project\" />
```

Comment:
> Is it that much necessary to revert this change?

Yes. This is a separate issue and it doesn't look like it blocks the merge for the optimization. If you think there is a problem with the project file, consider opening a separate issue with a clear explanation. Perhaps with VS screen shots because, for example, "polluting solution explorer with a "ghost" item" means nothing specific to me. Then, if you would like to propose a fix, submit a separate PR with it.